src/style.css

```
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  --theme-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-container__img {
+  display: block;
+  margin: 0 auto 45px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-container__card-header {
+  font-weight: 500;
+  line-height: 15px;
+  font-size: 12px;
+  margin: 0 auto 6px;
+}
+
+.product-container__code {
+  padding-bottom: 16px;
+}
+
+.rating__container {
+  display: flex;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 20px;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.star:last-child {
+  background-image: url("images/star.svg");
+  margin-right: 0;
+}
+
+.price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-bottom: 20px;
+}
+
+.price-container__price {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.buy-link {
+  display: flex;
+  padding: 12px 58px;
```

```suggestion
  justify-content: center;
  align-items: center;
  height: 40px;
```
Let's not use paddings here and use the properties of flexbox for centering the text. It will fix your tests